string checksum
string configuration_name

psen_scan_v2/ZoneSet[] zonesets